### ISSUE TITLE
SA1413 triggered for last item in list.

### ISSUE THREAD

**Issue by Jero9999** [DotNetAnalyzers/StyleCopAnalyzers#2416]
> warning SA1413: Use trailing comma in multi-line initializers

is being triggered for the last item in the list. For example the below code triggers the warning for the `new QuoteTitle_MustBeSet()` line.

```
static List<IValidateCommand<Quote>> OnCreateValidationCommands = new List<IValidateCommand<Quote>>
{
	new QuoteTitle_MustBeSet()
};
```

Running version stylecop 1.1.0-beta003

**Comment by sharwell** [comment]
Yes, this is a new warning for StyleCop Analyzers 1.1.0: #1121

This rule is part of our ongoing effort to choose rules not just for their single-user benefit, but for improving the effectiveness of long-term team development efforts.

I'll submit a pull request to update the documentation for the rule with additional background information.
